### PR TITLE
Adds two new clothes, relaxed slacks and circual sunglasses

### PR DESCRIPTION
**Pull request by EveLed**
## About The Pull Request

I decided to set aside an entire day and figure out how to do basic sprite work for ss13. These two clothing items are (hopefully) the fruits of my labor.
## Why It's Good For The Game

I think they look alright, nothing too special. Everybody likes more clothing options. I'm not gonna go ahead and say that these are the best looking sprites, but I think they mesh well with other clothing options

## Testing Photographs and Procedure

<details>

image
image
image
image

</summary>

</details>

## Changelog
:cl:
add: added circular sunglasses
add: added relaxed slacks
add: added circular sunglasses to the clothing vendor
add: added circular glasses to the clothes vendor
add: added relaxed slacks to the autodrobe
imageadd: sprites for circular sunglasses
imageadd: sprites for relaxed slacks
/:cl:

### CODE REVIEW

File: code/modules/vending/clothesmate.dm
Comment (HowToLoLu) [inline code/modules/vending/clothesmate.dm:104]: Probably should amend the CL with this change to be thorough

File: code/modules/clothing/glasses/_glasses.dm
Comment (HowToLoLu) [inline code/modules/clothing/glasses/_glasses.dm]: No need to redefine these 3 since they're already defined with these values on the parent type

**Review by HowToLoLu — COMMENTED** [review]
Looks great! There's only a couple of things I found that you may want to consider

File: code/modules/vending/clothesmate.dm
Comment (EveLed) [reply code/modules/vending/clothesmate.dm:104]: question, what does that mean exactly?

Comment (EveLed) [reply code/modules/vending/clothesmate.dm:104]: oh amend the Change log, okay

**Comment by HowToLoLu** [comment]
Looks like there's a temporary outage affecting a rust package provider that's breaking the Test workflow

**Comment by HowToLoLu** [comment]
> add: added circular sunglasses to the clothing vendor

I was refering to the normal set of circular glasses you added to the clothes vendor

**Comment by HowToLoLu** [comment]
Probably could be safe to reattempt bumping it now with another empty commit.

**Review by HowToLoLu — APPROVED** [review]
The code looks good as far as I can see, though I'm not a maintainer and you'll need to wait for one to review the PR.

**Comment by HowToLoLu** [comment]
Remember to bump the tests by pushing an empty commit; it's less likely someone will look at this to review with failing tests.

**Review by PestoVerde322 — APPROVED** [review]

**Review by Penwin0 — APPROVED** [review]

**Pull request merged**
